### PR TITLE
Add GET timezones endpoint, add ianaTimeZone field to User

### PR DESCRIPTION
**Pull request by nkharabaruk**
- Added a **GET** time_zones endpoint
- Add the `ianaTimeZone` field to the **User**

### CODE REVIEW

File: src/main/java/org/zendesk/client/v2/model/TimeZone.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+/**
+ * @author nkharabaruk
+ * @since 22/06/2023 22:00
+ */
+public class TimeZone {
+
+    private String translatedName;
+    private String name;
+    private String ianaName;
+    private String url;
```
Comment (PierreBtz) [inline src/main/java/org/zendesk/client/v2/model/TimeZone.java:16]: I don't see this field in the documentation, but maybe I missed it, it's a bit of a pain to navigate... Do you have a link I can check?

**Review by PierreBtz — CHANGES_REQUESTED** [review]
Thanks a lot for the contribution, I'd just like to double check the documentation of the `url` field before moving this further.

File: src/main/java/org/zendesk/client/v2/model/TimeZone.java (hunk 1 above)
Comment (nkharabaruk) [reply src/main/java/org/zendesk/client/v2/model/TimeZone.java]: The API call returns timezones with the`url` field. 
I also don't see it in the documentation: https://developer.zendesk.com/api-reference/apps/apps-support-api/all_locations/#time-zone-object
✔️ removed

**Review by PierreBtz — APPROVED** [review]
lgtm, thanks for your contribution!

**Pull request merged**